ansible_base/jwt_consumer/hub/auth.py

```diff
@@ -4,45 +4,42 @@
 
 from ansible_base.jwt_consumer.common.auth import JWTAuthentication
 from ansible_base.jwt_consumer.common.exceptions import InvalidService
+from ansible_base.rbac.models import RoleDefinition, RoleUserAssignment
 from ansible_base.resource_registry.models import Resource
 
 logger = logging.getLogger('ansible_base.jwt_consumer.hub.auth')
 
 
 class HubJWTAuth(JWTAuthentication):
 
-    def get_galaxy_models_and_functions(self):
+    def get_galaxy_models(self):
         '''This is separate from process_permissions purely for testability.'''
         try:
             from galaxy_ng.app.models import Organization, Team
-            from pulpcore.plugin.util import assign_role, remove_role
         except ImportError:
             raise InvalidService("automation-hub")
 
-        return Organization, Team, assign_role, remove_role
+        return Organization, Team
 
     def process_permissions(self):
         # Map teams in the JWT to Automation Hub groups.
-        Organization, Team, assign_role, remove_role = self.get_galaxy_models_and_functions()
+        Organization, Team = self.get_galaxy_models()
         self.team_content_type = ContentType.objects.get_for_model(Team)
         self.org_content_type = ContentType.objects.get_for_model(Organization)
 
-        orgs = []
-        teams = []
-        groups = []
-        for role_name in self.common_auth.token.get('object_roles', {}).keys():
-            if role_name.startswith('Org'):
-                for object_index in self.common_auth.token['object_roles'][role_name]['objects']:
-                    org_data = self.common_auth.token['objects']['organization'][object_index]
-                    ansible_id = org_data['ansible_id']
+        # TODO - galaxy does not have an org admin roledef yet
```

Comment:
Thought this wasn't true, but looks like it is:

https://github.com/ansible/galaxy_ng/blob/b1c841132c4d3d27c3d0ae5ce5b744a656ef4bab/galaxy_ng/app/settings.py#L435-L437